src/data/staking-products.json

```
@@ -190,6 +190,33 @@
         "eventAction": "Clicked",
         "eventName": "Clicked Vouch + Dirk go to link"
       }
+    },
+    {
+      "name": "AVADO",
+      "svgPath": "avado-glyph.svg",
+      "hue": 239,
+      "launchDate": "2018-12-01",
+      "url": "https://ava.do/",
+      "audits": [],
+      "minEth": 16,
```

Good to see support for Rocket Pool... 
I know this isn't fully utilized on the front-end yet, but similar to the Rocket Pool entry we should include the additional deposit requirements in the JSON here.
```suggestion
      "minEth": 16,
      "additionalStake": "10",
      "additionalStakeUnit": "% RPL",
```